### DISCUSSION TITLE
How can I get the current default values?

### DISCUSSION THREAD

**Discussion by jcready** [react-hook-form/react-hook-form#11190] [Q&A]
Since RHF only uses the `defaultValues` on the first render and the default values can be changed imperatively with `reset()` how can I find the form's **current** default values?

**Comment by Moshyfawn** [comment]
```js
const { formState } = useForm({ defaultValues })

const { defaultValues: currentDefaultValues } = formState
```

https://react-hook-form.com/docs/useform/formstate#:~:text=has%20interacted%20with.-,defaultValues,-object

**Reply by jcready** [reply]
That property doesn't exist in my version 7.9.0 and of course the documentation is unversioned. So even though I'm on the same major version this feature and the documentation is not useful/helpful to me.

**Reply by Moshyfawn** [reply]
Well, we can't backport the feature to the older version, mate, can we?

**Reply by jcready** [reply]
Consider versioning the documentation and/or indicate for each property which version it was introduced like the nodejs docs. The current major version was released 2.5 years ago so the amount of change that has occurred in the minor version releases is substantial.

**Reply by Moshyfawn** [reply]
That's fair. But the project is also open-source, so consider contributing 😉
